soc/arm/nordic_nrf/nrf51/CMakeLists.txt

```diff
@@ -1,4 +1,7 @@
 zephyr_sources(
-  power.c
   soc.c
   )
+
+zephyr_sources_ifdef(CONFIG_SYS_POWER_MANAGEMENT
```

Comment:
Isn't there a case where someone wants to use power.c _without_ defining SYS_POWER_MANAGEMENT?

Reply:
Arghhhh. I have missed the sys_arch_reboot(). One moment.

Reply:
Fixed